### DISCUSSION TITLE
How to prevent MethodUp/MethodDown actions from adding to jumplist?

### DISCUSSION THREAD

**Discussion by Wkjlovec** [JetBrains/ideavim#1075] [Q&A]
I'm using IdeaVim and want to prevent the MethodUp/MethodDown(binding to nnoremap [m or nnoremap ]m) actions from being added to the jumplist when navigating between methods. Currently, every method navigation is recorded in the jumplist, making CTRL-O/CTRL-I navigation cumbersome.
Any help would be appreciated.

**Comment by TARRAHADNAN** [comment]
Jo soy. Le. Comprardo li oltima lisa con la configeration

[image: ADNAN TARRAH] <https://www.utac.com/>
*ADNAN TARRAH*
*Electrician & ITmanager*

  Mobile : +212641997607
  Email : ***@***.***
***@***.***?subject=Devis&body=Bonjour%2C%20Adnan%20Tarrah>

[image: Linkedin] <https://www.linkedin.com/in/adnan-tarrah-706a7b1b2>[image:
Facebook] <https://www.facebook.com/tarrah.adnan/>[image: Twitter]
<https://twitter.com/TARRAHADNAN18>[image: Instagram]
<https://www.instagram.com/electro.adnan/?hl=fr>  tarrah.adnan
Morocco, Beni mellal,
23000
***@***.***,-6.3756338,12z/data=!4m5!3m4!1s0x0:0xce829b01459c166c!8m2!3d32.3371487!4d-6.3392401>


Le sam. 28 déc. 2024 à 09:56, Kangjie Wu ***@***.***> a
écrit :
